testutil/testproxy/relayerproxy.go

```diff
@@ -304,8 +304,8 @@ func MarshalAndSend(
 		Header: http.Header{
 			"Content-Type": []string{"application/json"},
 		},
-		URL:  &url.URL{Scheme: scheme, Host: proxiedServices[proxyServeName].Host},
-		Host: proxiedServices[proxyServeName].Suppliers[serviceId].Hosts[0],
+		URL:  &url.URL{Scheme: scheme, Host: proxiedServices[proxyServeName].ListenAddress},
+		Host: proxiedServices[proxyServeName].Suppliers[serviceId].PubliclyExposedEndpoints[0],
```

Comment:
When do we ever use not `[0]`?

Reply:
https://github.com/pokt-network/poktroll/pull/477#discussion_r1562461950